avocado/__init__.py

```diff
@@ -12,6 +12,8 @@
 # Copyright: Red Hat Inc. 2013-2014
 # Author: Lucas Meneghel Rodrigues <lmr@redhat.com>
 
+from __future__ import absolute_import
```

Comment:
It certainly does look like unnecessary boilerplate code. Looking for some bug reports in other projects though, it seems like it's a safety net against unexpected import behavior, example:

https://github.com/inducer/pudb/issues/194

So... yeah, I still don't have an opinion about that. I'll sleep on it.

Reply:
One other approach, that I was trying to avoid on the basis of extra complexity and work, was to define a different set of lint checks for examples/*.  But, it looks like it'd bring the best outcome.

Reply:
Since we only support Python >= 2.7 and >= 3.4, we don't need these two warnings.  The explanation is given here: 

https://www.python.org/dev/peps/pep-0328/#rationale-for-absolute-imports